### ISSUE TITLE
java.lang.ClassNotFoundException: Didn't find class "com.bumptech.glide.Glide"

### ISSUE THREAD

**Issue by Md-Mudassir47** [mieszko4/react-native-zoom-us#190]
@mieszko4 
Updated to latest version `6.7.0` and followed the new changes in `AndroidManifest.xml` file. The build was successful but it crashes on launch and throws the below error.
```
FATAL EXCEPTION: main
Process: com.careerlabs, PID: 12742
java.lang.NoClassDefFoundError: Failed resolution of: Lcom/bumptech/glide/Glide;
	at us.zoom.sdk.ZoomSDK.initialize(ZoomSDK.java:93)
	at ch.milosz.reactnative.RNZoomUsModule$2.run(RNZoomUsModule.java:163)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:224)
	at android.app.ActivityThread.main(ActivityThread.java:7562)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
Caused by: java.lang.ClassNotFoundException: Didn't find class "com.bumptech.glide.Glide" on path: DexPathList[[zip file "/data/app/com.careerlabs-51QRNydJzC5DIH6hsKNr3g==/base.apk"],nativeLibraryDirectories=[/data/app/com.careerlabs-51QRNydJzC5DIH6hsKNr3g==/lib/arm64, /data/app/com.careerlabs-51QRNydJzC5DIH6hsKNr3g==/base.apk!/lib/arm64-v8a, /system/lib64, /system/product/lib64]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:230)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
	... 9 more

```

### Version Details
`"react-native": "^0.66.3",`

**Comment by Md-Mudassir47** [comment]
I was able to fix it by adding a dependency inside `/android/app/build.gradle`

```js
dependencies {

    //zoom glide issue fix
    implementation 'com.github.bumptech.glide:glide:4.12.0'
...
}
```
There was one more issue, i.e when clicking on the 4 dots icon (as shown in the screenshot) the app was crashing but adding the above dependency fixed this problem aswell. 

![dd530406-dedb-4837-88cf-448de85b44cc](https://user-images.githubusercontent.com/85115578/152532475-cc8eab32-9cd9-4fbf-bed6-fc691656545d.jpeg)

**Comment by mieszko4** [comment]
Awesome. Would you like to create a pr with your fix?

**Comment by Md-Mudassir47** [comment]
> Awesome. Would you like to create a pr with your fix?

Absolutely, I will have to mention it under the Installation steps right? inside Readme.

**Comment by mieszko4** [comment]
> > Awesome. Would you like to create a pr with your fix?
> 
> Absolutely, I will have to mention it under the Installation steps right? inside Readme.

Actually, I've done that already - it could to be added in the lib's dependency. I've released the fix in v6.7.1. Thnx again!
